modules/tracktion_engine/utilities/tracktion_Engine.h

```diff
@@ -33,15 +45,15 @@ class Engine
    #endif
     static juce::Array<Engine*> getEngines();
 
-    TemporaryFileManager& getTemporaryFileManager() const;
-    AudioFileFormatManager& getAudioFileFormatManager() const;
-    PropertyStorage& getPropertyStorage() const;
-    UIBehaviour& getUIBehaviour() const;
-    EngineBehaviour& getEngineBehaviour() const;
-    DeviceManager& getDeviceManager() const;
-    MidiProgramManager& getMidiProgramManager() const;
-    ExternalControllerManager& getExternalControllerManager() const;
-    RenderManager& getRenderManager() const;
+    TemporaryFileManager& getTemporaryFileManager() const; ///< Get the shared temporary file manager allowing to handle default app and user temporary folders.
+    AudioFileFormatManager& getAudioFileFormatManager() const; ///< Get the audio files format manager that maintains a list of available wave file formats.
+    PropertyStorage& getPropertyStorage() const; ///< Get the top level User settings customizable xml file.
+    UIBehaviour& getUIBehaviour() const; ///< Get the engine UI elements customization helper class allowing to change default ui engine elements.
+    EngineBehaviour& getEngineBehaviour() const; ///< Get engine customization helper class allowing to change the engine default behavior.
+    DeviceManager& getDeviceManager() const; ///< Get the top level class for handling audio / midi devices @see juce::AudioDeviceManager.
+    MidiProgramManager& getMidiProgramManager() const; ///< Get the top level class that handles midi banks, programs, sets or presets.
```

Comment:
MIDI should always be all-caps.

There's probably lots of lower-case uses of it in the library, but should get this right in all new content, and fix old ones when we see them (only in text, not necessarily in the code itself)

Reply:
Applied suggested changes, thanks for helping me improve my (non native ) english too.